chapters/introduction.tex

```diff
@@ -1,27 +1,36 @@
-keeping in mind that you are not going to be the only person
-collecting it, using it, or looking back on it.
-We hope to provide you two key tools by the time you finish this book.
-First, we want you to form a mental model of data collection as a ``social process'',
+This book is intended to teach all users of development data 
+how to handle data effectively, efficiently, and ethically. 
+An empirical revolution has changed the face of research economics rapidly over the last decade. 
+%had to remove cite {\cite{angrist2017economic}} because of full page width
+Economics graduate students of the 2000s expected to work with primarily ``clean'' data from secondhand sources. 
+Today, especially in the development subfield, working with raw data- 
+whether collected through surveys or acquired from ``big'' data sources like sensors, satellites, or call data records- 
+is a key skill for researchers and their staff. 
+At the same time, the scope and scale of empirical research projects is expanding: 
+more people are working on the same data over longer timeframes. 
+As the ambition of development researchers grows, so too has the complexity of the data
+on which they rely to make policy-relevant research conclusions. 
+Yet there are few guides to the conventions, standards, and best practices 
+that are fast becoming a necessity for empirical research.
+This book aims to fill that gap, providing guidance on how to handle data efficiently, transparently and collaboratively. 
+
+This book is targeted to everyone who interacts with development data: 
+graduate students, research assistants, policymakers, and empirical researchers. 
+It covers data workflows at all stages of the research process: design, data acquisition, and analysis.
+This book is not sector-specific; it will not teach you econometrics, or how to design an impact evaluation. 
+There are many excellent existing resources on those topics. 
+Instead, this book will teach you how to think about all aspects of your research from a data perspective, 
+how to structure research projects to maximize data quality, 
+and how to institute transparent and reproducible workflows. 
+The central premise of this book is that data work is a ``social process'',
 in which many people need to have the same idea about what is to be done, and when and where and by whom,
-so that they can collaborate effectively on large, long-term projects.
-Second, we want to provide a map of concrete resources for supporting these processes in practice.
-As research teams and timespans have grown dramatically over the last decade,
-it has become inefficient for everyone to have their own personal style
-dictating how they use different functions, how they store data, and how they write code.
+so that they can collaborate effectively on large, long-term research projects.
+It aims to be a highly practical resource: each chapter offers code snippets, links to checklists and other practical tools, 
```

Comment:
This is not true "_each chapter offers code snippets_" and I am not sure how it can be achieved.

Reply:
indeed. changed in 266ab5e03d086737554039a3ffeddacb347a29fb